hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/SpanTypeAttributeEnricher.java

```diff
@@ -76,74 +72,6 @@
   private final String envoyIngressSpanValue = RawSpanConstants.getValue(Envoy.ENVOY_INGRESS_SPAN);
   private final String envoyEgressSpanValue = RawSpanConstants.getValue(Envoy.ENVOY_EGRESS_SPAN);
 
-  @Override
```

Comment:
Just curious, why the method shift? Can we keep it here and have the utility methods at the bottom of the class, for better readability of the actual logic in the class?

Reply:
Bad formatting change. Moved the method to the original place.